brands/shop/beauty.json

```diff
@@ -116,5 +116,15 @@
       "name": "Yves Rocher",
       "shop": "beauty"
     }
+  },
+  "shop/beauty|Perfect Brows NYC": {
+    "countryCodes": ["us"],
+    "matchNames": ["Perfect Brows"],
+    "tags": {
+      "beauty": "eyebrows;waxing;eyelash;henna tattoo;facials;men's services;",
```

Comment:
We probably want to stick with the common known beauty tags here for this to be useful in search / display later

Reply:
I agree w @tas50 ..

@amous4822 can you check this list here for commonly used `beauty=*` tags:
https://taginfo.openstreetmap.org/keys/beauty#values

Reply:
@tas50 @bhousel I did check. Some of the tags I've added have never been used 😅. I will edit it. Thank you.

Reply:
Thanks! looks good 👍